### PR TITLE
Fix: Fix linting errors

### PR DESCRIPTION
**Pull request by lightbringer1991**

### CODE REVIEW

File: src/components/adslot-ui/InformationBox/index.spec.jsx [hunk 1]
```diff
@@ -4,11 +4,10 @@ import { shallow } from 'enzyme';
 
 describe('InformationBoxComponent', () => {
   it('should render with props', () => {
-    const component = shallow(<InformationBox
-      title="render title here"
-      icon="assets/img#done"
-      children="<div>I am child</div>"
```
Comment (lightbringer1991) [inline src/components/adslot-ui/InformationBox/index.spec.jsx]: `children` should not be a prop. See `react/no-children-prop`

**Comment by lightbringer1991** [comment]
No report found because the codecov didn't run in master due to travis failure

File: src/components/adslot-ui/InformationBox/index.spec.jsx (hunk 1 above)
Comment (sonhanguyen) [reply src/components/adslot-ui/InformationBox/index.spec.jsx]: I don't think it even worked

**Comment by omgaz** [comment]
Not sure why code-cov is saying there's no base report (assume it's because of previous failures). But I'll merge this.

**Pull request merged**